src/profile-logic/processed-profile-versioning.js

```diff
@@ -818,5 +818,39 @@ const _upgraders = {
       thread.stringArray = stringTable.serializeToArray();
     }
   },
+  [18]: profile => {
+    // funcTable gains a new field: columnNumber.
+    for (const thread of profile.threads) {
+      const { funcTable, stringArray } = thread;
+      const stringTable = new UniqueStringArray(stringArray);
+      funcTable.columnNumber = [];
+      for (
+        let funcIndex = 0;
+        funcIndex < thread.funcTable.length;
+        funcIndex++
+      ) {
```

Comment:
I think we miss some cases where we need to add `null` to `funcTable.columnNumber[funcIndex]`. This is less error-prone to do this operation once here:
```js
funcTable.columnNumber[funcIndex] = null; 
```

Then below we can remove all `else` blocks and just keep the code that adds another value.